### DISCUSSION TITLE
Happy 8 yrs Backpack for Laravel

### DISCUSSION THREAD

**Discussion by rjjrbatarao** [Laravel-Backpack/community-forum#1070] [General]
I've recently purchased the pro 399 deal with a great discount!!, this is to support this Awesome project. I've been a backpack user since the 3.x version on my personal hobby projects, during that time I was exploring the web development world while working on a non software field, I stumbled across this project and my first impression of using this was a amazing which stucked on my head for years even with new similar projects like Filament came after I'll still recommend backpack for everyone for a professional and highly customizable Admin dashboard. Goodjob and more years to come Backpack team.

**Comment by jcastroa87** [comment]
Hi @rjjrbatarao ,

Thank you so much for your wonderful message and for purchasing the Pro! We're thrilled to have your support for our project.

It's incredibly heartening to hear about your journey with Backpack since the 3 version. Knowing that Backpack has been a part of your web development exploration and that it left such a lasting impression on you is truly inspiring. We're honored that you continue to recommend Backpack.

Your kind words about our admin dashboard mean a lot to our team. We're committed to delivering the best tools and support to our users, and feedback like yours encourages us to keep pushing forward.

If you ever have any questions, need assistance, or have suggestions, please don't hesitate to reach out. We're here to help you make the most out of Backpack.

Thank you again for your support and for being part of our community. Here's to many more successful years together!

I will ping @tabacitu to he read this amaizing message.

Cheers.

**Comment by tabacitu** [comment]
Thank you so much for purchasing, and for taking the time to write this message @rjjrbatarao 🙏
I super appreciate it, you made my day. 

Thank you!
